examples/README.adoc

```diff
@@ -0,0 +1,33 @@
+
+== Examples and Tutorial
+
+Welcome to the Hawkular Alerting Examples and Tutorials.  This will hopefully get you quickly up and running.  In addition there are the official docs:
+
+* http://www.hawkular.org/community/docs/developer-guide/alerts.html[User Guide]
+* http://www.hawkular.org/docs/rest/rest-alerts.html[REST API Guide]
+* http://github.com/hawkular/hawkular-alerts[Github]
+
+If you you have questions please find us on IRC at irc://irc.freenode.org/#asciidoctor[#hawkular on freenode].
```

Comment:
typo: you you have -> you have
typo: irc://irc.freenode.org/#asciidoctor instead #hawkular room